Python Tutorial Exercises(Week 1)/#1.py

```diff
@@ -0,0 +1,11 @@
+""" Week 1 - Exercise 1 
```

Comment:
Only this file has this title, which is inconsistent to the rest, please add it to the rest of the script files

Reply:
Scratch that, just remove this line from this file, and that will have everything else consistent.